### PR TITLE
feat: Add axum framework guide

### PR DESCRIPTION
**Pull request by xinnks**

### CODE REVIEW

File: sdk/rust/guides/axum.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Turso + Axum
+sidebarTitle: Axum
+description: Set up Turso in your Axum project in minutes
+---
+
+![Tauri banner](/images/guides/axum-banner.png)
```
Comment (notrab) [inline sdk/rust/guides/axum.mdx:7]:
```suggestion
![Axum banner](/images/guides/axum-banner.png)
```

File: sdk/rust/guides/axum.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+</Steps>
+
+## Examples
+
+<CardGroup cols={2}>
+  <Card
+    title="Turso + Axum Todo List"
+    icon="github"
+    href="https://github.com/turso-extended/app-turso-axum-todo"
```
Comment (notrab) [inline sdk/rust/guides/axum.mdx:97]: Should this not be inside `tursodatabase/examples`?

**Review by notrab — APPROVED** [review]
Few comments but lgtm. Nice one! 🥳

**Comment by xinnks** [comment]
@notrab All issues fixed!

**Pull request merged**